src/UserSpaceInstrumentation/OrbitUserSpaceInstrumentation.cpp

```diff
@@ -134,8 +134,8 @@ bool& GetIsInPayload() {
 
 }  // namespace
 
-void StartNewCapture() {
-  current_capture_start_timestamp_ns = CaptureTimestampNs();
+void StartNewCapture(uint64_t capture_start_timestamp_ns) {
+  current_capture_start_timestamp_ns = capture_start_timestamp_ns;
 
   // If the library has just been injected, initialize the
```

Comment:
If we get rid of the call to `AttachAndStopNewThreadsOfProcess`, and https://github.com/google/orbit/pull/3809 goes for another kind of initialization, should we just remove this whole comment?